boards/arm/atsamd21_xpro/atsamd21_xpro.dts

```diff
@@ -46,6 +54,14 @@
 	clock-frequency = <48000000>;
 };
 
+&tcc0 {
+	status = "okay";
+	compatible = "atmel,sam0-tcc-pwm";
+	/* Gives a maximum period of 1.4s */
```

Comment:
TCC0 has 24 bits of resolution so the maximum period is 48 MHz / 2**24 / 1024 = 357 s.

The samples require a minimum period of 1s, so you can safely reduce the prescaler to 4.

Reply:
ah, yes. forgot that TCC0 is one with 24 bits. Fixed.